handlers/server.go

```diff
@@ -56,6 +58,8 @@ func AddFlags(fs *pflag.FlagSet, s *WebHookServer) {
 	fs.StringVar(&c.Repo, "repo", c.Repo, "refers to the project repo address")
 	fs.StringVar(&c.GitHubToken, "github-token", c.GitHubToken, "contains the githubtoken info")
 	fs.StringVar(&c.WebhookSecret, "webhook-secret", c.WebhookSecret, "contains the webhooksecret key")
+	fs.StringVar(&c.TravisCIToken, "travis-ci-token", c.TravisCIToken, "contains Travis-CI access token to trigger the PR build")
+	fs.StringVar(&c.TravisRepoAccount, "travis-ci-repo-name", c.TravisRepoAccount, "contains repo address of Travis-ci build Ex: kubeedge/kubeedge")
```

Comment:
repo account or repo name? I assume it's repo name. So, neither travis-ci prefix is not required nor flag name should be repoName instead of repoAccount?

Reply:
@m1093782566 Yes will change the description

Reply:
> This flag is for specify repo account or repo name? I assume it's repo name. So, neither travis-ci prefix is not required nor flag name should be repoName instead of repoAccount?

@m1093782566 , modified as suggested